src/clj/puppetlabs/services/jruby/jruby_core.clj

```diff
@@ -142,43 +142,43 @@
 ;;; Public
 
 (schema/defn ^:always-validate
-  initialize-config :- jruby-schemas/JRubyPuppetConfig
+  initialize-config :- jruby-schemas/JRubyConfig
   [config :- {schema/Keyword schema/Any}]
-  (-> (get-in config [:jruby-puppet])
+  (-> (get-in config [:jruby])
```

Comment:
We have a ticket for "Refactor[ing] jruby-puppet config..." (SERVER-1189), and so I had assumed that for that we would do some refactoring of how this config works and integrates with Puppet Server. Even if we don't end up doing that ticket, IMO it makes more sense to leave this as-is right now and then see what we have to do for Puppet Server integration.